test/restaurant_inspections_api/handlers/counties_test.clj

```diff
@@ -0,0 +1,13 @@
+(ns restaurant-inspections-api.handlers.counties-test
+  (:require [restaurant-inspections-api.handlers.counties :refer :all]
+            [clojure.test :refer :all]))
+
+(def ctx-mock {:request {:params {}}})
+(def response-mock {:meta {}
+                    :data [{:district "D1"
+                            :countynumber 23
+                            :countyname "Dade"
+                            :inspections 20630}]})
+;(deftest handle-ok-test
```

Comment:
yeah there's no use in testing handle-ok for counties. There's no real logic. We can actually remove the commented test. We can leave the file and mocks just in case we add more behavior later.